engine/scripts/audit.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+cd "$(dirname "$0")"
```

Comment:
I don't understand this "$0" it that the first input parameter given to the script? So change dir, why is that needed?

Reply:
Without this the script runs in the context of the `dir` it was called from. This ensures it runs in the context of the `dir` it is located in.

From https://fedingo.com/how-to-set-current-working-directory-to-directory-of-shell-script/